src/include/stir/KOSMAPOSL/KOSMAPOSLReconstruction.h

```diff
@@ -208,12 +211,14 @@ class KOSMAPOSLReconstruction:
   shared_ptr<TargetT> kpnorm_sptr;
  //kernel parameters
   int num_neighbours,num_non_zero_feat,num_elem_neighbourhood,num_voxels,dimz,dimy,dimx;
+  int stop_iterative_kernel_at_iter_num;
```

Comment:
I agree with @KrisThielemans  that "stop" is confusing because it doesn't actually stop using the kernel. `freeze_iterative_kernel_at_iter_num` might be clearer